composer.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "roave/you-are-using-it-wrong",
+    "description": "Composer plugin enforcing strict type checks in downstream package dependants",
+    "type": "composer-plugin",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Marco Pivetta",
+            "email": "ocramius@gmail.com"
+        }
+    ],
+    "require": {
+        "php":                       "^7.3.0",
+        "ext-json":                  "*",
+        "composer-plugin-api":       "^1.0.0",
+        "ocramius/package-versions": "^1.4.0",
+        "vimeo/psalm":               "dev-master"
+    },
+    "require-dev": {
+        "phpunit/phpunit":          "^8.0.0",
+        "infection/infection":      "^0.12.2",
+        "composer/composer":        "^1.8.5",
+        "doctrine/coding-standard": "^6.0.0",
+        "symfony/process":          "^4.2.8",
+        "psalm/plugin-phpunit": "^0.5.5"
```

Comment:
All of these are lined up, except for `"psalm/plugin-phpunit"`.

You could also use https://github.com/localheinz/composer-normalize

Reply:
Since this is a plugin itself, I will avoid requiring `localheinz/composer-normalize` here, as it may introduce hidden complexity in the local setup, which may hide further plugin infrastructure issues. Will align manually for now.